app/services/clients/adminusers.client.js

```diff
@@ -673,6 +673,26 @@ module.exports = function (clientOptions = {}) {
     )
   }
 
+  const updateMerchantUrl = (serviceExternalId, merchantUrl, correlationId) => {
```

Comment:
We could use the `updateService` method here https://github.com/alphagov/pay-selfservice/blob/master/app/services/service.service.js#L44 rather than needing to add an additional client method.
This was introduced more recently, so not all places where we patch a service haven't been updated to use this, but using it would reduce the amount of code required.

Reply:
Thanks - I have removed the `updateMerchantUrl` method.